pkg/services/baremetal/host/host_test.go

```diff
@@ -1652,3 +1656,60 @@ var _ = Describe("actionProvisioned", func() {
 		}),
 	)
 })
+
+var _ = Describe("actionProvisioned SSHAfterInstallImage=false", func() {
+	It("test reboot annotation for SSHAfterInstallImage=false", func() {
```

Comment:
this formulation is not how it is intended. What I can suggest, is to split this up to a BeforeEach and smaller "It" functions that each have a specific purpose, e.g. "It reboots the server via hardware reboot".

Reply:
@janiskemper I created three `It()` functions. BeforeEach was not needed. Each has a slightly different setup.